ci/testgrid/README.md

```diff
@@ -1,8 +0,0 @@
-# Testgrid config
-
-This directory contains the config for our
-[Testgrid](https://github.com/kubernetes/test-infra/tree/master/testgrid)
-instance.
-
-- `Makefile` Commands to interact with the Testgrid instance regarding updates.
-- `config.yaml` Testgrid configuration.
```

Comment:
Don't want to include this in another README?

Reply:
Indeed. Done.